blueman/gui/manager/ManagerDeviceMenu.py

```diff
@@ -168,13 +168,32 @@ def on_device_property_changed(self, lst: "ManagerDeviceList", _device: Device,
             if key in ("Connected", "UUIDs", "Trusted", "Paired"):
                 self.generate()
 
+    def _translate_error_message(self, msg: str) -> str:
+        msg = msg.split(":", 3)[-1].strip()
+
+        # https://sourceware.org/git/?p=glibc.git;a=blob;f=sysdeps/gnu/errlist.h
+        # https://git.musl-libc.org/cgit/musl/tree/src/errno/__strerror.h
+        # https://git.uclibc.org/uClibc/tree/libc/string/_string_syserrmsgs.c
+        if msg == "Protocol not available":
+            # ENOPROTOOPT
+            return _("No audio endpoints registered")
+        elif msg in ("Input/output error", "I/O error"):
+            # EIO
+            return _("Input/output error")
```

Comment:
Does "Communication error or problem"make sense here? If so how about using that 😄

Reply:
EIO is used for all sorts of shit. Just grep for it in the BlueZ source and you'll find >120 locations where it gets returned and I think about half of them can end up here as a DBus connection error. I'd say the meaning really is just that *something* went wrong more or less unexpectedly.

Occurences in our GitHub issues oftentimes have to do with firmware issues and the like.

I guess the best advice we could give is to check the bluetoothd logs as there's typically a warning or error level message that hints at what actually went wrong. Not sure if and how I'd place such an advice here, though. I wouldn't put it in this message, but we could think about a "More", "Help" or whatever link that gives some advice like that. For ENOPROTOOPT it could be to install the pulseaudio Bluetooth module, possibly even naming distribution specific package names but that sounds more like something that should go to a Wiki page we could link to.

Reply:
👀 ok then 😄. How about we keep it like it is but log something more helpful and refer users to that, logging.warning("Something unexpected happened, possible causes are.....")?

Reply:
Good idea. Added. :+1: